src/components/WalletDetails/walletDetails.hook.tsx

```diff
@@ -0,0 +1,10 @@
+// this is temporary once we setup a service for it, we will remove it.
+
+export const useUserWallet = () => {
+  return {
```

Comment:
> are we sure the userWalletDetails won't be undefined?

referencing to my above comment i see this wont be undefined for now as this is hardcoded but I assume this will come from an API it would be great to handle it now itself later on it might skip through the mind for adding the check